types/passport-facebook/index.d.ts

```diff
@@ -27,40 +31,62 @@ export interface AuthenticateOptions extends passport.AuthenticateOptions {
     authType?: string | undefined;
 }
 
-export interface StrategyOption {
+export interface StrategyOptions {
```

Comment:
Made plural to follow conventions in passport-oauth2 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/48ea2d936c47ae6cb56b7b0719d0fa2cd30a9612/types/passport-oauth2/index.d.ts#L100